front/app/containers/Admin/dashboard/visitors/VisitorsOverview.tsx

```diff
@@ -58,15 +56,15 @@ const VisitorsOverview = ({ uniqueVisitorDataDate }: Props) => {
           onProjectFilter={handleProjectFilter}
           onChangeResolution={setResolution}
           showAllTime={false}
-          minDate={uniqueVisitorDataDate}
+          minDate={moment(START_DATE_SESSION_DATA_COLLECTION)}
```

Comment:
For the visitors dashboard, the minimum date you can select will be the date we started collecting visitor data, back in 2022.